src/util/delta_util.ts

```diff
@@ -39,7 +39,7 @@ from |----------------------------------------| to (t1)
                         from |-------------------| to (t1)
 */
 export function hasIntersection(t1: TimeFilter, t2: TimeFilter): boolean {
-  return t1.from < t2.to && t1.from > t2.from; // t1.windowSize === t2.windowSize
+  return t1.from < t2.to && t1.from >= t2.from; // t1.windowSize === t2.windowSize
```

Comment:
it's not easy to find the most value change for this release

Reply:
"Es sind die kleinen Dingen.." 😂